lib/gui/app/components/source-selector/source-selector.tsx

```diff
@@ -220,12 +356,28 @@ export class ImageSelector extends React.Component<
 			return;
 		}
 
-		const source = new sdk.sourceDestination.File({
-			path: imagePath,
-		});
+		let source;
+		if (SourceType === sourceDestination.File) {
+			source = new sourceDestination.File({
+				path: imagePath,
+			});
+		} else {
+			if (!_.startsWith(imagePath, 'http')) {
```

Comment:
Please explicitly check for 'http://' or 'https://'.
This code won't throw for `httpwhatever://invalid-url`

Reply:
It will throw as an invalid image so this code is actually redundant for the purpose of a validity check.
I'm just adding it as a way to instruct users, because it throws anyway